script/bootstrap.sh

```diff
@@ -99,11 +72,14 @@ if [ $(uname) = "Linux" ]; then
                          clang-format-6.0 \
                          cmake \
                          ninja-build \
+                         python3-setuptools \
+                         python3-pip \
                          lcov -y
 
-    ## Install cmake 3.16.1
+    ## Install newest CMake
     match_version $(cmake --version | egrep -o '[0-9].*') ${MIN_CMAKE_VERSION} || {
-        install_cmake "3.16.1"
+        pip3 install -U pip
+        pip3 install -U cmake
```

Comment:
If path is not setup correctly, users can still end up using wrong `cmake`. Should we check the `cmake` version after installation and warn the user loudly if version is not satisfied?